### ISSUE TITLE
Статика

### ISSUE THREAD

**Issue by k0t3n** [marazmiki/django-selectel-storage#9]
Есть ли поддержка сохранения/раздачи django static в библиотеке?

**Comment by marazmiki** [comment]
Сохранение и раздача – это о ``django.contrib.staticfiles``, я полагаю?  Если да, то оно использует под капотом механизм django storages, соответственно, статику можно хранить раздавать любым совместимым методом, включая и это приложение.

**Comment by k0t3n** [comment]
Увы, у меня не получилось настроить через данную библиотеку - статика всегда пыталась раздаваться с localhost. 

Обнаружил, что у Селектела есть S3-similar API, поэтому получилось завести через [django-storages](https://github.com/jschneier/django-storages), модифицируя конфиги.

**Comment by marazmiki** [comment]
> Обнаружил, что у Селектела есть S3-similar API, поэтому получилось завести через [django-storages](https://github.com/jschneier/django-storages), модифицируя конфиги.

Да, я видел анонс о поддержке совместимого с S3  API хранилища Селектела. Но в бою испытывать не доводилось, не считая негативного опыта попытки загрузки файла через ``s3put`` (не взлетело).

> у меня не получилось настроить через данную библиотеку - статика всегда пыталась раздаваться с localhost.

А не могли бы Вы пояснить, что именно пошло не так? Это помогло бы исправить проблемы и не допускать их в дальнейшем.

**Comment by k0t3n** [comment]
@marazmiki при установке `STATICFILES_STORAGE` статика всё равно отдается с текущего хоста, т.к. из storage отдается относительная ссылка из-за пустого `storage_url`, получаемого при авторизации. При этом сам метод авторизации при запросе статики не вызывается.
Несмотря на это`manage.py collectstatic` отрабатывает правильно, проблема только в отдаче.

Reproducible env: python3.8, django==3.0.5

**Comment by marazmiki** [comment]
У какая версия ``django-selectel-storage``?

**Comment by k0t3n** [comment]
Тестировалось на master https://github.com/marazmiki/django-selectel-storage/commit/0acf17be030d7f303f6761c210aa463610047ee6

**Comment by marazmiki** [comment]
У меня при обычном запросе индексной страницы, в которой есть ``{% static "..." %}`` получилось как-то так:

<img width="871" alt="Скриншот 2020-04-20 13 52 20" src="https://user-images.githubusercontent.com/207900/79722462-4d6d2200-830e-11ea-8f45-39ec6db38724.png">

Т.е. класс стореджа всё-таки инициализируется и, соответственно, происходит ``Auth``

**Comment by k0t3n** [comment]
Я проверял на django admin (собственно, отдавать статику по админке и была основной задачей). В вашей конфигурацией с ней всё тоже ок?

**Comment by marazmiki** [comment]
Не знаю, админки не было :)

Но сейчас попробую устроить демо-проект

**Comment by marazmiki** [comment]
@k0t3n, однако Вы правы, имеет место быть косяк. Спасибо за сигнал. Исправлю в ближайшее время

**Comment by marazmiki** [comment]
Seems 2b fixed
